docs/features.md

```diff
@@ -82,6 +82,7 @@ Global plugin settings and settings defined at project level are merged together
 * `diagnostics_gutter_marker` `"dot"` *gutter marker for code diagnostics: "dot", "circle", "bookmark", "cross" or ""*
 * `show_code_actions_bulb` `false` *show a bulb in the gutter when code actions are available*
 * `show_color_box` `true` *Show a box next to the color if the language server provides color support*
+* `disabled_capabilities`, `[]` *Turn off client capabilities (features), like "hover" or "completion"
```

Comment:
Please clarify that valid identifiers are coming from `TextDocumentClientCapabilities`.

Reply:
Good point, added all valid options here. Currently the subset is fairly small, so I don't want to mislead people into thinking all the capabilities can be toggled this way.